Makefile

```diff
@@ -12,7 +12,12 @@ MD2ROFF_BIN=github.com/davidnewhall/md2roff@v0.0.1
 RSRC_BIN=github.com/akavel/rsrc
 
 # If upx is available, use it to compress the binaries.
-UPXPATH=$(shell which upx)
+# Skip upx in Mac ARM environments: https://github.com/upx/upx/issues/446
+ifeq ($(shell uname -ps),Darwin arm)
+	UPXPATH=
+else
+	UPXPATH=$(shell which upx)
```

Comment:
Instead of an `else`, can you put line 15 back, and remove lines 18 and 19? Thanks!

Reply:
Good call!